fs/src/jsMain/kotlin/externals/kotlin-multiplatform-projects-fs/version.txt

```diff
@@ -0,0 +1 @@
+2, 0.5.7
```

Comment:
What is the purpose of this file?

Reply:
not sure. it was generated by dukat along with `inputs.txt` but looking into it now my guess is they're only needed by dukat to determine whether to regenerate the externals. i'll delete them